### DISCUSSION TITLE
Option to prevent cloud backup of downloaded audiobooks

### DISCUSSION THREAD

**Discussion by pendletong** [LeoKlaus/plappa#185] [Feature Requests]
Just noticed that my iCloud backup of my iPhone contained all the downloaded audiobooks. I can't see any real reason for that but at the very least it should be optional.

**Comment by LeoKlaus** [comment]
Hi Gereth, thank you for reporting this! That is absolutely not intentional, so I converted this to an issue (#186).

The simplest workaround until I implement a fix would be to just disable iCloud backup for plappa on your end.

I will close this discussion now and keep you updated at #186.
